src/audio/QGCAudioWorker.cpp

```diff
@@ -187,10 +187,18 @@ bool QGCAudioWorker::_getMillisecondString(const QString& string, QString& match
     return false;
 }
 
-QString QGCAudioWorker::_fixMillisecondString(const QString& string) {
+QString QGCAudioWorker::_fixTextMessageForAudio(const QString& string) {
     QString match;
     QString newNumber;
     QString result = string;
+    //-- Look for modes
+    if(result.contains("POSCTL", Qt::CaseInsensitive)) {
```

Comment:
Seems like we should do this sort of thing in the firmware instead of fighting a never ending battle in QGC.